docs/Paymaster/paymastermethods.md

```diff
@@ -113,6 +113,14 @@ const feeQuotesResponse = await biconomyPaymaster.getPaymasterFeeQuotesOrData(us
 - In token list we can specify a list of addresses for the choices in tokens we want to have our users pay in
 - We can also decide to choose a preferred token for the response to include
 
+:::warning
+
+**_Important:_** When using **Token Paymaster** with ERC20 tokens, always ensure to calculate the **feeQuote** correctly. This is crucial to avoid transaction reverts due to insufficient token balance after execution. The `feeQuote` should consider both the transaction cost and any other **token** movements within the same operation.
```

Comment:
Update this
"... due to insufficient token balance after execution. "

to
".. due to insufficient token balance when the EntryPoint calls postOp on Paymaster after calldata execution."